src/views/docs-view/server.ts

```diff
@@ -31,13 +31,15 @@ import {
 // Local imports
 import { getProductUrlAdjuster } from './utils/product-url-adjusters'
 import { SidebarProps } from 'components/sidebar'
-import { EnrichedNavItem } from 'components/sidebar/types'
 import { getBackToLink } from './utils/get-back-to-link'
 import { getDeployPreviewLoader } from './utils/get-deploy-preview-loader'
 import { getCustomLayout } from './utils/get-custom-layout'
 import type { DocsViewPropOptions } from './utils/get-root-docs-path-generation-functions'
 import { getStaticPathsFromAnalytics } from 'lib/get-static-paths-from-analytics'
 import { withTiming } from 'lib/with-timing'
+import outlineItemsFromHeadings from 'lib/docs/outline-items-from-headings'
+import { DocsViewProps } from './types'
+import { GetStaticPaths, GetStaticProps, GetStaticPropsResult } from 'next'
```

Comment:
⛏️ could we lift this to the top of the file?

Reply:
ah yep! I rely on auto-imports in VSCode a little heavily 😅 I wonder if we could add import re-ordering to pre-commit formatting or whatnot 🤔 I might look into that when I have bandwidth!

Reply:
✅ updated to re-order imports, matched the existing `Third-party` / `HashiCorp` / `Global` / `Local` sections 👍